tests/config_files/test_config_map_to_map.yaml

```diff
@@ -1,17 +1,17 @@
 data:
   n_pix: 224
-  psize: 2.146 
+  psize: 2.146
   submission:
     fname: tests/data/dataset_2_submissions/test_submission_0_n8.pt
     volume_key: volumes
     metadata_key: populations
     label_key: id
   ground_truth:
-    volumes: tests/data/Ground_truth/test_maps_gt_flat_10.pt 
-    metadata: tests/data/Ground_truth/test_metadata_10.csv 
-  mask: 
+    volumes: tests/data/Ground_truth/test_maps_gt_flat_10.pt
+    metadata: tests/data/Ground_truth/test_metadata_10.csv
+  mask:
     do: true
-    volume: data/Ground_truth/mask_dilated_wide_224x224.mrc
+    volume: tests/data/Ground_truth/mask_dilated_wide_224x224.mrc
```

Comment:
Remove tests from here, as it's the mask for the regular results. It agrees with the directories on OSF.